launcher-implementation/src/main/scala/xsbt/boot/Configuration.scala

```diff
@@ -76,6 +76,7 @@ object Configuration {
   def DefaultVersionPart = "Default"
   def DefaultBuildProperties = "project/build.properties"
   def SbtVersionProperty = "sbt.version"
+  def SbtAppProperty = "sbt.app"
```

Comment:
The setting to control this is called `name` in sbt, so to me it makes sense to call this `sbt.name`.

Reply:
Ok, I've moved ahead to use `sbt.name`, still not 100% convinced on this choice, since `app.name` (`sbt`) is used for the other settings (e.g. `"sbt".version`), in this case we are dynamically re-defining one value that is used all the way down in configuration, but here you have ...

Reply:
forgot to mention that with last changes `sbt.boot.properties` should have this entry:
```
name: ${sbt.name-read(sbt.name)[sbt]}
```